downloads/swan-lake-release-notes/swan-lake-alpha/RELEASE_NOTE.md

```diff
@@ -0,0 +1,1432 @@
+
+Integers intValue = 5;
+Integers intArray = [1, 2, 3, 4];
+Integers intMap = {i: 1, j: 2};
+Integers integers = [intValue, intArray, intMap];
+```
+
+##### Updated Syntax for User-Defined Error Constructions
+
+The error constructor expression now requires the `error` keyword to construct a user-defined error.
+
+**Previous Syntax:**
+```ballerina
+MyError myError = MyError("Message");
+```
+ 
+**New Syntax:**
+```ballerina
+MyError myError = error MyError("Message");
+```
+
+#### Changes on Casting with Errors
+
+Errors cannot be cast away (i.e., if the value that is being cast can be an error, the type to which the cast is attempted should also have a subtype of error). 
+
+The following was allowed previously.
+
+```ballerina
+function foo() returns string|int|error {
+    return error(“Error Message”);
+}
+
+public function main() {
+    string s = <string>foo(); // Cast without considering the error.
+}
+```
+
+This is now disallowed and can be rewritten as follows.
+
+```ballerina
+function foo() returns string|int|error {
+    return error("Error Message");
+}
+
+public function main() {
+    string s = <string>checkpanic foo(); // use`checkpanic` to panic if `foo()` returns an error and then attempt the cast
+}
+```
+
+1. Using `checkpanic` - This will preserve the `panic` behavior (but with a different error - the previous error was a cast error. Now, it will be the actual error returned by `foo()`).
+
+    ```ballerina
+    function foo() returns string|int|error {
+        return error("Error Message");
+    }
+
+    public function main() {
+        string s = <string>checkpanic foo(); // use`checkpanic` to panic if `foo()` returns an error and then attempt the cast
+    }
+    ```
+
+2. Using `check` - Alternatively, if the function’s return type allows returning the error, `check` can be used before attempting the cast to return the error instead of panicking.
+
+```ballerina
+function foo() returns string|int|error {
+    return error(“Error Message”);
+}
+
+public function main() returns error? {
+    string s = <string>check foo(); // use`check` to return the error if `foo()` returns an error, and then attempt the cast
+}
+```
+
+##### Changes on `toString` and `toBalString` with Errors
+
+It was previously possible to call `toString()` and `toBalString()` on unions of errors and non-errors.
+
+```ballerina
+function print(any|error val) {
+    string s = val.toString();
+}
+```
+
+This has now been disallowed and the error scenarios need to be handled explicitly. The above example can be rewritten as follows.
+
+```ballerina
+function print(any|error val) {
+    string s = val is error ? val.toString() : val.toString();
+}
+```
+
+#### Changes on Object Type Inclusion with Qualifiers
+
+- When object type inclusion is used with an object type descriptor with qualifiers (`isolated`, `client`, `service`), it is now mandatory for the object being included also to have these qualifiers.
```

Comment:
This change is wrong!

As mentioned in the doc, this has to be
```suggestion
- When object type inclusion is used with an object type descriptor with qualifiers (`isolated`, `client`, `service`), it is now mandatory for the including object to have these qualifiers.
```

Reply:
This is grammatically incorrect.  "Including" cannot be used as an adjective.

@Samallama

Reply:
The spec uses "including type". There are few other places in this release note that uses the same too.

Reply:
If this has to be changed it should probably be something like "the object in which the inclusion is done".

Reply:
+1 Changed to that. Thanks!